packages/graphile-build-pg/src/plugins/PgMutationCreatePlugin.js

```diff
@@ -44,6 +51,13 @@ export default (function PgMutationCreatePlugin(
           .filter(table => !!table.namespace)
           .filter(table => table.isSelectable)
           .filter(table => table.isInsertable)
+          .filter(
+            table =>
+              pgEnableDefaultMutationTables.length == 0 ||
+              pgEnableDefaultMutationTables.indexOf(
+                `${table.namespace.name}.${table.name}`
+              ) >= 0
```

Comment:
I'm afraid this might be slightly too simple; e.g. it's possible to define tables that contain a `.` in their name, and if a table contains capital letters then people should wrap the table name in quotes otherwise the case-insensitivity of PostgreSQL will drop the capitals. 

I recommend you use the `parseTypeIdentifier` method from PGJWTPlugin.js:

https://github.com/graphile/graphile-build/blob/d2da24040a6a352679b5985b4829634f32396e7a/packages/graphile-build-pg/src/plugins/PgJWTPlugin.js#L120-L134

to extract the namespace and table names (which you should do just before the `builder.hook` call so it only has to be evaluated once at server start time) then check these during the filter.